src/Compilers/Extension/project.json

```diff
@@ -1,8 +1,12 @@
 ﻿{
   "dependencies": {
     "EnvDTE": "8.0.0",
-    "Microsoft.VisualStudio.Shell.14.0": "14.3.25407",
-    "Microsoft.VisualStudio.Shell.Interop.10.0": "10.0.30319"
+    "Microsoft.VisualStudio.Shell.15.0": "15.0.26014-alpha",
```

Comment:
Understand the reasoning but disagree based on experience here.  Mixing official and unofficial packages is a nightmare because of the dependency drag.  Until the VS SDK team can publish a *full* and *comprehensive* SDK there is no reasonable way for us to depend on bits and pieces of it.  That is part of the follow up discussion we are having with them in a few weeks.